docs/source/interfaces/Tomographic_Reconstruction.rst

```diff
@@ -347,8 +347,10 @@ You can run local reconstructions as well, however that requires properly settin
 - External python interpretor path, for example:
   - C:/Anaconda/python.exe for Windows
```

Comment:
These should be a sub-list and that still isn't rendering correctly. Also backtick the directories